src/components/prescriptions/PrescriptionAddComponent.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+    <div
+        class="prescriptionBox"
+        @submit.prevent="addTest">
+        <h3 class="prescriptionBox__header">Dodaj receptę:</h3>
+        <form class="prescriptionBox__form prescriptionForm">
+            <div>
+                <div class="inputBox">
+                    <label for="nameAndSurnameDoctor">
+                        Imię i nazwisko lekarza:</label
+                    >
+                    <select v-model="nameAndSurnameDoctor">
+                        <option
+                            disabled
+                            value="">
+                            Wybierz lekarza
+                        </option>
+                        <option
+                            v-for="research in getDoctorsArray()"
+                            :key="research.registryId">
+                            {{ research.doctorName }}
+                        </option>
+                    </select>
+                </div>
+                <div class="inputBox">
+                    <label for="medicine">Lek:</label>
+                    <select v-model="medicine">
+                        <option
+                            disabled
+                            value="">
+                            Wybierz lek
+                        </option>
+                        <option
+                            v-for="research in getMedicineArray()"
+                            :key="research.medicinalProductId">
+                            {{ research.medicineName }}
+                        </option>
+                    </select>
+                </div>
+
+                <div class="inputBox">
+                    <label for="frequency">Częstotliwość:</label>
+                    <input
+                        id="title"
+                        v-model="frequency"
+                        type="text" />
+                </div>
+
+                <div class="inputBox">
+                    <label for="researchDate"
+                        >Wybierz dzień wystawienia recepty:</label
+                    >
+                    <input
+                        id="addManually"
+                        v-model="researchDate"
+                        type="date" />
+                </div>
+            </div>
+
+            <div class="actionsAndInfo">
+                <button-component
+                    :disabled="disableAddTest"
+                    wide
+                    >Zapisz receptę</button-component
+                >
+                <bouncing-balls-component :visible="isLoading" />
+                <error-info :message="error" />
+                <span
+                    v-show="successFlag"
```

Comment:
non semantic name of aria. Is it aria label describes passwords matching process? No. You have to change it because its is wrong for user with a11y problems. His/her assistive tool read "passwords match" and what about then? He/she is confused bacause info is about success data send and assistive touch read "passwords match". Please dont copy code blindly without thinking what you copy :(